blog/2015-03-12.Debugging-an-Externally-Launched-Wildfly.adoc

```diff
@@ -0,0 +1,97 @@
+= Debugging an Externally launched WildFly
+:page-layout: blog
+:page-author: robstryker
+:page-tags: [jbosstools, devstudio, server, wildfly]
+
+Debugging externally-launched Java applications has always been possible in Eclipse via the `Remote Java Application` launch configuration type, so long as you manually add the host and debug port to the launch configuration.  A few years ago, you may have link:http://docs.jboss.org/tools/whatsnew/core/core-news-3.3.0.M3.html[seen an enhancement] we here at JBoss Tools provided, making it easier to connect your sourcecode in Eclipse to an externally-launched process. 
+
+Well, history tends to repeat itself, and more enhancements are in the pipeline that make this even easier. 
```

Comment:
pipeline as in "not there yet" ? which version would this be available in ? should let the reader know.